test/toucan/db_test.clj

```diff
@@ -169,9 +170,29 @@
 
 ;; TODO - Test simple-delete!
 
-;; TODO - Test simple-insert-many!
+;; Test simple-insert-many!
+(expect
+ '(4 5)
```

Comment:
you can just do `[4 5]` here instead of a `'(4 5)` since expectations will consider sequences with the same contents to be the same even if they're vectors instead of lists